src/material-examples/cdk-datepicker-filter/cdk-datepicker-filter-example.css

```diff
@@ -0,0 +1 @@
+/** No CSS for this example */
```

Comment:
I think it would make sense to combine these three examples into one that's like "Restricting available dates"